xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -45,48 +45,8 @@ void CEdl::Clear()
   m_lastCutTime = 0;
 }
 
-bool CEdl::ReadEditDecisionLists(const CFileItem& fileItem, const float fFrameRate, const int iHeight)
+bool CEdl::ReadEditDecisionLists(const CFileItem& fileItem, const float fFramesPerSecond)
```

Comment:
While you are at it: It does not make sense to declare by value parameters with primitve types as `const`.

Reply:
I did notice that but decided it was for reasons beyond my understanding 😉

Will fix it up shortly.

Reply:
Although, would this not signify that this value is not modified in this method?

Reply:
keep it, not a real issue, more a matter of taste. sorry for the noise.

Reply:
indeed, see discussion in slack.

Reply:
NP